### DISCUSSION TITLE
Welcome to WAS Node Discussions!

### DISCUSSION THREAD

**Discussion by WASasquatch** [WASasquatch/was-node-suite-comfyui#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Request ideas for nodes
  * Share ideas or workflows
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by atheran2** [comment]
Hello there! Amazing work so far. I recently started with T2I generation and being used to UE5 and Houdini's node systems, I found CUI to be a better option for me compared to A1111, even if it is still lacking behind. For now I don't do much with it besides some tests to learn the ropes, but what I want from it eventually is to start making a comic.

One thing I can't understand is how to use Tensor Batch to Image, I would appreciate any more information about it because it looks like something that could save me time compared to upscaling every generation, then again, I might understand it completely wrong.

As for ideas for nodes, the two that come to mind that would help a lot with spaghetti nodes are an on/off switch to halt the execution at some point so it doesn't continue down the controlnet path for example if I don't want it to and a "portal" node, that accepts any input or number of inputs and outputs them somewhere else that the node is used. I could make a mockup of it, if you don't understand what I described.

**Reply by WASasquatch** [reply]
Tensors Image to Batch takes a batched generation and allows you select a single image from the batch. So say you did a batch of 6, you can select one of those images with ID 0-5 where 0 is the 1st image.

I will be putting a load batch images node in soon. Little busy today so probably tonight. It will allow you grab all images in a directory. From here it will either output a single image from batch by incremental ID like Tensor Batch to Image or, hopefully, I will be able to get incremental batching in. So say you load 30 images, and set comfyui to run 30 times, it will incremental through all images loaded as output on each run, allowing essentially batch runs with different images. 


 have logic nodes ready, but wildcard inputs are not working in ComfyUI yet, so I couldn't release. For example I have input switch by boolean, so it would send ether input_a or input_b depending on if the boolean is 1 or 0.
